corehq/apps/export/static/export/js/models.js

```diff
@@ -181,9 +181,12 @@ Exports.ViewModels.ExportColumn.prototype.isCaseName = function() {
     return this.item.isCaseName();
 };
 
+Exports.ViewModels.ExportColumn.prototype.translatedHelp = function() {
```

Comment:
You can declare Exports as a global for this file by putting this at the top of the file:
```javascript
/*global Exports*/
```
as described [here](https://github.com/dimagi/js-guide/blob/master/linting.md#global-variables)

(also fine to just ignore for now)